python/tests/test_datetime.py

```diff
@@ -1,31 +1,33 @@
-#
-import nvstrings
+# Copyright (c) 2018-2019, NVIDIA CORPORATION.
+
 import numpy as np
+import pandas as pd
+
+import nvstrings
+
+from utils import assert_eq, initialize_rmm_pool
+
+
+initialize_rmm_pool()
+
+
+def test_timestamp2int():
+    s = nvstrings.to_device(["2019-03-20T12:34:56Z", "2020-02-29T23:59:59Z"])
+    s1 = pd.Series(["2019-03-20T12:34:56Z", "2020-02-29T23:59:59Z"]).apply(
+        lambda x: pd.Timestamp(x))
+    got = s.timestamp2int()
+    expected = s1.astype(np.int64)
+    assert np.allclose(got, expected, 10)
+
+    s1 = pd.Series(["2019-03-20T12:34:56Z", "2020-02-29T23:59:59Z"]).apply(
+        lambda x: pd.Timestamp(x, unit='ms'))
+    got = s.timestamp2int(units='ms')
+    expected = s1.astype(np.int64)
+    assert np.allclose(got, expected, 10)
+
 
-#
-from librmm_cffi import librmm as rmm
-from librmm_cffi import librmm_config as rmm_cfg
-rmm_cfg.use_pool_allocator = True
-rmm.initialize()
-
-s = nvstrings.to_device(["2019-03-20T12:34:56Z","2020-02-29T23:59:59Z"])
-print(s)
-print(".timestamp2int()",s.timestamp2int())
-print(".timestamp2int(ms)",s.timestamp2int(units='ms'))
-
-s = nvstrings.to_device(["04/24/2019","02/29/2020"])
-print(".timestamp2int(M/D/Y)",s.timestamp2int(format='%m/%d/%Y'))
```

Comment:
Seems we are missing a bunch of testcases here.

Reply:
Agreed, this is a minimal test case. We absolutely will want to create tests for the other formats noted in `test_datetime`.

Reply:
Are we comfortable with only the minimal format test for this initial PR, or do we need to expand a bit?